config/routing_config_template.json

```diff
@@ -0,0 +1,35 @@
+// Hex numbers are strings prefixed with "0x"
+{
+    "net_id_aliases": {
+        "name": "0x000009",
+        "another_name": 1337
+    },
+    "routing": [
```

Comment:
Could we name this `uplink` or something so that it makes more sense?

Reply:
Yes, I was trying to think of better keys. 
How do you feel about `eui` and `devaddr`?